pykechain/exceptions.py

```diff
@@ -15,30 +14,54 @@ class APIError(Exception):
     :ivar detail: details of the error
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, text, *args, **kwargs):
```

Comment:
are we sure when we add `text` that this is compatible with before. Normally in django / DjangoRestFramework we have 'detail' or 'message' or 'messages' which is of `Union[Text,Sized,Dict]`

Reply:
Not sure we need this, args[0] might work just as well (with a check if there are any arguments)